backend/distributor/filters.py

```diff
@@ -0,0 +1,24 @@
+from django_filters import Filter, FilterSet
+
+from .models import Distribution
+
+
+class MultipleListFilter(Filter):
+    def filter(self, qs, value):
+        if not value:
+            return qs
+
+        self.lookup_expr = 'in'
+        values = value.split(',')
+        return super(MultipleListFilter, self).filter(qs, values).distinct()
```

Comment:
Let's set some limit.

Reply:
What limit should I put there? Maybe `10`?

Reply:
10 is okay.